admin_manual/installation/letsencrypt.rst

```diff
@@ -55,71 +56,77 @@ Without explicit denying by command, certbot will auto-update on each run.
   sudo apt-get update
   sudo apt-get install certbot
 
-To update certbot, you need to manually run
+To update Certbot, you need to run the following commands manually:
 
 ::
 
   sudo apt-get update
   sudo apt-get upgrade
   sudo apt-get dist-upgrade
 
-To run certbot type
+To run Certbot use the following command:
 
 ::
 
   sudo /usr/bin/certbot
 
-**Register your eMail address**
+**Register your email address**
 
-Run the certbot once to prepare the environment. You should register your eMail 
-address for urgent renewal and security notifications. Use the command based on
-your installation method.
+Run Certbot once to prepare the environment.
+Use the command based on your installation method.
 
 ::
 
-  sudo /opt/letsencrypt/certbot-auto register -email <your-eMail-address>
+  sudo /opt/letsencrypt/certbot-auto register -email <your-email-address>
+
+.. note::
+   You should register your email address for urgent renewal and security notifications.
 
 Onetime preparations
 --------------------
 
-Create a Letsencrypt challenge directory
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Create a Let's Encrypt challenge directory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-It is assumed that your webserver root is in ``/var/www``. Adopt this path if different.
+It is assumed that your webserver root is ``/var/www``.
+Use its path if it is different from this.
 
 ::
 
   cd /var/www
   sudo mkdir letsencrypt
   sudo chown root:www-data letsencrypt
 
-Create Letsencrypt config files
+Create Let's Encrypt config files
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- Create following files in the Letsencrypt directory. They will help to maintain your certfificates.
-- Replace the path to certbot and the certbot script name according to your installation.
-- Make all files executable except ``cli.ini`` by running ``sudo chmod +x <script-name>``
-- All scripts have to be executed with ``sudo``
+- Create following files in the Let's Encrypt directory. They will help to maintain your certificates.
+- Replace the path to Certbot and the Certbot script name based on your installation.
+- Make all files executable except ``cli.ini`` by running ``sudo chmod +x <script-name>``.
+
+.. note::
+   All scripts have to be executed with ``sudo``.
 
 ::
 
-	cd /etc/Letsencrypt
+    cd /etc/Let's Encrypt
```

Comment:
we both have a small typo... it is a directory created by certbot. needs fixing

mine:  Letsencrypt  --> letsencrypt
yours: Let's Encrypt --> letsencrypt